ava.config.mjs

```diff
@@ -0,0 +1,20 @@
+const config = {
```

Comment:
Please make these changes inside `ava.config.js`. We do not want to use an `mjs` file.